client/app/bundles/course/assessment/question/scribing/reducers/scribingQuestionReducer.jsx

```diff
@@ -0,0 +1,77 @@
+import actionTypes from '../constants';
+
+export const initialState = {
+  question: {
+    id: null,
+    title: '',
+    description: '',
+    staff_only_comments: '',
+    maximum_grade: 0,
+    weight: 0,
+    skill_ids: [],
+    skills: [],
+    published_assessment: false,
+    attempt_limit: null,
+  },
+  isLoading: false,
+  save_errors: undefined,
+};
+
+function apiReducer(state, action) {
+  const { type } = action;
+  switch (type) {
+    case actionTypes.FETCH_SCRIBING_QUESTION_REQUEST:
+    case actionTypes.CREATE_RESPONSE_REQUEST:
+    case actionTypes.UPDATE_SCRIBING_QUESTION_REQUEST:
+    case actionTypes.SUBMIT_FORM_LOADING: {
+      const { isLoading } = action;
+      return {
+        ...state,
+        is_loading: isLoading,
+        save_errors: undefined,
+      };
+    }
+    case actionTypes.FETCH_SCRIBING_QUESTION_SUCCESS:
+    case actionTypes.CREATE_SURVEY_QUESTION_SUCCESS:
+    case actionTypes.UPDATE_SCRIBING_QUESTION_SUCCESS:
+    case actionTypes.SUBMIT_FORM_SUCCESS: {
+      const { question } = action.data;
+      question.maximum_grade = parseInt(question.maximum_grade, 10);
```

Comment:
It feels more correct to do data conversion before the data actually reaches the reducer. An API issue?